lib/warehouse/clients/assembly/connection.ex

```diff
@@ -43,7 +43,7 @@ defmodule Warehouse.Clients.Assembly.Connection do
 
   @impl true
   def handle_info({:gun_up, _, _, _, _}, _state) do
-    Logger.debug("Warehouse.Clients.Assembly.Connection connected")
+    Logger.info("Warehouse.Clients.Assembly.Connection connected")
```

Comment:
Good idea to always display these messages.

Reply:
Yes for now, until we can change the log level live